pyrokinetics/kinetics/KineticsReaderJETTO.py

```diff
@@ -33,10 +36,13 @@ def read(self, filename: PathLike) -> Dict[str, Species]:
             psi = psi - psi[0]
             psi_n = psi / psi[-1]
 
-            rho = kinetics_data["RMNMP"][-1, :].data
+            Rmax = kinetics_data["R"][-1, :].data
+            Rmin = kinetics_data["RI"][-1, :].data
+
+            r = (Rmax - Rmin) / 2
+            rho = r / r[-1]
```

Comment:
We need to check to see if the last closed flux surface is actually the last element. I think data stop just shy of it so we may need to slightly tweak it